docs/networking.md

```diff
@@ -156,14 +156,16 @@ To enable this mode in a cluster, with Calico as the CNI and Network Policy prov
 
 ```
   networking:
-    calico: {}
+    calico:
+      apiVersion: v3
 ```
 
 You will need to change that block, and add an additional field, to look like this:
 
 ```
   networking:
     calico:
+      apiVersion: v3
```

Comment:
should this just be `version`?

Reply:
Let's leave it as is. If anything `calicoAPIVersion` might be less ambiguous, if you're up for making that change.

Reply:
Looking at this now, it might be best represented as `majorVersion`?